responsibleai_text/tests/common_text_utils.py

```diff
@@ -123,7 +127,15 @@ def load_blbooks_genre_dataset():
     dataset = g.first().sort_index().reset_index().drop_duplicates()
     dataset = dataset[dataset.label != 1].reset_index(drop=True)
     dataset = dataset.drop(columns=grouping_col)
-    print(dataset)
+    return dataset
+
+
+def load_dbpedia_dataset():
+    dataset = datasets.load_dataset(
+        "DeveloperOats/DBPedia_Classes", split="test")
+    dataset = pd.DataFrame({"text": dataset["text"], L1: dataset[L1]})
```

Comment:
Instead of lines 136 to 138, I'd recommend simply renaming the column from L1 to DBPEDIA_LABEL, like

`
dataset.rename(columns={L1 : DBPEDIA_LABEL}, inplace=True)
`

Reply:
good idea, done!